Base/Python/slicer/parameterNodeWrapper/serializers.py

```diff
@@ -1052,39 +1057,48 @@ def create(type_):
         return None
 
     @staticmethod
-    def _serializerName(name: str) -> str:
-        return f"{name}.serializer"
+    def _typeName(name: str) -> str:
+        return f"{name}.type"
 
     @staticmethod
     def _valueName(name: str) -> str:
         return f"{name}.value"
 
     def _getValueSerializer(self, parameterNode, name: str) -> Serializer:
-        type_ = pickle.loads(bytes.fromhex(self._typeSerializer.read(parameterNode, self._serializerName(name))))
-        return createSerializerFromAnnotatedType(type_)
+        modulename, classname = self._typeSerializer.read(parameterNode, self._typeName(name))
+        if (modulename, classname) == ("builtins", "NoneType"):
+            # "from builtins import NoneType" doesn't work
+            return createSerializerFromAnnotatedType(type(None))
+        else:
+            mod = sys.modules[modulename]
```

Comment:
:+1: Definitely less concerned about security issues with `pickle` gone.

Although I wonder if here it would be better to use `importlib.import_module` instead of fetching cached modules? It seems unlikely but not impossible for the module to not yet be imported. I don't expect there to be security issues as `import_module` requires the module to already exist on-disk; it can't execute code stored in the scene.

Reply:
Will update to use `importlib`

Reply:
Updated.